src/data/ua_overrides.js

```diff
@@ -1210,6 +1210,26 @@ const AVAILABLE_UA_OVERRIDES = [
       },
     },
   },
+  {
+    /*
+     * Bug 1842767 - UA override for passport.bilibili.com
+     *
+     * Spoofing as Chrome makes the login page use a mobile layout.
+     */
+    id: "bug1842767",
+    platform: "android",
+    domain: "passport.bilibili.com",
+    bug: "1842767",
+    config: {
+      matches: [
+        "*://*.passport.bilibili.com/",
```

Comment:
Nit: `*` [matches zero or more characters](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_scripts#globs), so we don't actually need this.